### DISCUSSION TITLE
Winget and Canonical Multipass

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#5047] [General]
Winget always reports there is an upddate to Canonical Multipass even though I have the latest version installed.  Latest version from Canonical is 1.14.1+win.  Winget says it should be 1.14.1.  I have updated via winget but get the same results when i run upgrade-available.  Minor bug but a little annoying.

**Comment by Trenly** [comment]
Will be resolved by https://github.com/microsoft/winget-pkgs/pull/197333
